src/test/java/org/carlspring/cloud/storage/s3fs/util/EnvironmentBuilder.java

```diff
@@ -68,14 +74,49 @@
         return env;
     }
 
+    /**
+     * Attempt to retrieve OS Environment Variable
+     * @return
+     */
+    private static String getEnvValue(@Nonnull String key)
+    {
+        String envKeyName = getS3EnvName(key);
+        String value = System.getenv(envKeyName);
+
+        return value;
+    }
+
+    /**
+     * @param key Non-prefixed environment name (i.e. bucket.name)
+     * @return Prefixed uppercase environment name (i.e. S3FS_BUCKET_NAME || S3FS_MINIO_BUCKET_NAME)
+     */
+    private static String getS3EnvName(@Nonnull String key)
+    {
+        // Sometimes the key will contain `S3FS_` or `s3fs.property.name` which is why we need to sanitize the string
+        // to ensure consistency.
+        String sanitized = StringUtils.removeStartIgnoreCase(key, "S3FS.");
+        sanitized = StringUtils.removeStartIgnoreCase(sanitized, "S3FS_");
+        sanitized = sanitized.replaceAll("\\.", "_").toUpperCase();
+
+        String integrationSuite = System.getProperty("running.it");
+        if(integrationSuite != null && integrationSuite.equalsIgnoreCase("minio"))
```

Comment:
```suggestion
        if (integrationSuite != null && integrationSuite.equalsIgnoreCase("minio"))
```

Reply:
ignoring this since this logic will be entirely removed in #186 for now.